### ISSUE TITLE
Unexpected token < if kirby-html-minifier is installed

### ISSUE THREAD

**Issue by gearsdigital** [gearsdigital/kirby-analytics-dashboard#5]
As reported in the [Kirby forums](https://forum.getkirby.com/t/kirby-analytics-dashboard/9408/21?u=gearsdigital), there is an javascript error if the [kirby-html-minifier](https://github.com/jenstornell/kirby-html-minifier) plugin is installed and enable. It says `Uncaught SyntaxError: Unexpected token <`.

**Comment by gearsdigital** [comment]
After little debugging I figured out what happens. The plugin uses a trivial delimiter `<` to extract html tags.

Under the hood it makes use of `Kirby\Component\Response` which builds and returns the response by various inputs. I don't know exactly why – but somehow the component seems to handle JS and CSS responses individually and I think it should exclude such types.

Currently there is no fix for that but I opened an PR over there which targets this issue.
